### ISSUE TITLE
Use both LDAP and taiga db authentication

### ISSUE THREAD

**Issue by muppeth** [ensky/taiga-contrib-ldap-auth#49]
Hi guys,
I'm wondering if it's possible to use both LDAP and and taigadb authentication. I would like to keep "Registration" on taiga open and still let my LDAP users to login without a need to create new account.

**Comment by martijnbastiaan** [comment]
@muppeth . I've implemented this in my fork: https://github.com/martijnbastiaan/taiga-contrib-ldap-auth/commit/3e4ff796551af0539a9082d8abb0e52edfd14541. 

@ensky It doesn't look like this project is maintained anymore. Would you like me to take it over?

**Comment by veox** [comment]
@martijnbastiaan Hey, that's cool! I'll try to integrate that [in a fork](https://github.com/Technariumas/taiga-contrib-ldap-auth) for an organisation I'm working for on this...

I also see you've used a few commits of mine, that's even cooler! :D May mean simple merging instead of cherry-picking.

**Comment by martijnbastiaan** [comment]
Yeah! I did want  your changes as well, so I figured I just built them on top. Do you (or your organization) want to take over the development for this plugin? I've tried contacting @ensky but he doesn't seem to respond. I would be willing to take it over as well, but it seems you've done more work on it than me so that's maybe not fair.

**Comment by veox** [comment]
This is more of a one-shot for me, getting it to work acceptably on a specific deployment. I can't promise to keep maintaining it for general use.

I say go ahead! :)

There is the tiny problem of `taiga-contrib-ldap-auth` package having already been registered with PyPI. Might need to change the project name, or request transfer (never tried that, not sure if possible). Tack an `-ng` at the back?..

**Comment by martijnbastiaan** [comment]
OK! I'll look at it this weekend hopefully!

**Comment by gthieleb** [comment]
@martijnbastiaan, @muppeth 

Would you accept pull request #53 in your branch? I would like to rely on the alternative db based login your branch offers. I have workarounded an issue that some SSL ciphersuites are dropped out in either Python 3.5 or current Redhat Release (7.3). Currently they are hardcoded but in a second iteration I would add additional `LDAP_SSL_VALIDATE`, `LDAP_SSL_VERSION`, `LDAP_SSL_CIPHERS` settings attributes. What do you think?

**Comment by madmath03** [comment]
@gthieleb It's an old subject but since the other projects seems stalled, I forked the project and unified the different PR, yours included.

https://github.com/Monogramm/taiga-contrib-ldap-auth-ext

Feel free to contribute if you see any improvements.
